site/templates/film/item.jet

```diff
@@ -62,94 +80,106 @@
           {{end}}
 
           {{yield awardNominations() film}}
-          <s72-element-switcher>
-            {{if isEnabled("element_switcher_enabled")}}
-              <div class="meta-detail-switcher-tagline">
-                <p>{{film.Tagline}}</p>
-              </div>
-            {{end}}
-            <div class="{{ isEnabled("element_switcher_enabled") ? "s72-hide" : "" }}"> {* hide class prevents FOUC *}
-              <div class="meta-detail-synopsis">{{film.Overview | raw}}</div>
-              <div class="meta-detail-cast">
-              {{if len(film.Cast) > 0 }}
-                <h2>{{i18n("meta_detail_cast_title")}}</h2>
+          <div class="element-switcher-sponsor">
+            <s72-element-switcher class="meta-detail-info-width">
+              {{if isEnabled("element_switcher_enabled")}}
```

Comment:
Nah, the tagline [has been there since the initial commit 2 years ago](https://github.com/shift72/core-template/blame/785534892740cfedb1d5d1fd60b7128f9813f517/site/templates/film/item.jet).

Reply:
Thats the metaItemTagline block which doesnt have the tagline in it anymore (lol)

[I added it back in in the element switcher PR](https://github.com/shift72/core-template/pull/267/files#diff-69356e56b7aeda8fec0deb76be7ccf24cdea865b0aadb8d5b0c3ec1537c6e17dR68)

Reply:
Ah my bad, it's confusing terminology - I saw "tagline" and was thinking of runtime etc, while `film.Tagline` is the "Short description" in Admin.